src/Test/Utilities/Portable/ModuleInitializerAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !NET5_0
```

Comment:
Referencing the .NET 5 TFM issue so we know to come back and change this if the design for what `NET5_0` represents changes

https://github.com/dotnet/sdk/issues/13377